sample_apps/run_example.sh

```diff
@@ -107,27 +132,95 @@ Steps=( "rm_non_git_files"
         "compile_app"
         "get_measurement_of_trusted_app"
         "author_policy"
-        "construct_policyKey_platform_is_trusted"
-        "construct_policyKey_measurement_is_trusted"
-        "combine_policy_stmts"
-        "print_policy"
-        "construct_platform_key_attestation_stmt_sign_it"
-        "print_signed_claim"
+            # These sub-step fns are subsumed under author_policy()
+            "construct_policyKey_platform_is_trusted"
```

Comment:
These sub-steps are subsumed (and run by) `author_policy`.

It is difficult to indicate through this array that user has to only run `author_policy` or run this list of sub-steps.

The script itself takes care of this duplication.

We can try to document it in the instructions.md or README.md, but I feel this business is very ver low-level.

And will be tedious to explain everything.

Comments welcome ... on how-to handle this nesting issue.

Reply:
Also see my comments in the diff for `setup_simple_app()` and `setup_simple_app_for_oe()` which explains how the execution handles these nested sub-step functions list.